### PR TITLE
create McAfee NSM modeling rules

### PR DESCRIPTION
**Pull request by guytamir10**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-6951

### CODE REVIEW

File: Packs/McAfeeNSM/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+
+# McAfee NSM (Network Security Manager)
+This pack includes Cortex XSIAM content.
+
+## McAfee NSM Syslog configuration
+*McAfee NSM syslog event types:*
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:6]:
```suggestion
McAfee NSM syslog event types:
```

File: Packs/McAfeeNSM/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+* IPS Events
+* Faults
+* User Activity (audit logs)
+
+*config Syslog IPS Events:*
+1. on McAfee NSM go to Manager -> Setup -> Notification -> IPS Events -> syslog.
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:12]:
```suggestion
1. In McAfee NSM, go to **Manager** > **Setup**> **Notification** > **IPS Events** > **syslog**.
```

File: Packs/McAfeeNSM/README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+2. click *YES* on "Enable Syslog Notification".
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:13]:
```suggestion
2. In Enable Syslog Notification, click **YES .
```

File: Packs/McAfeeNSM/README.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+3. bellow click on *+* and add Target server.
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:14]:
```suggestion
3. Click *+* and add the target server. 
```

File: Packs/McAfeeNSM/README.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+*McAfee NSM syslog event types:*
+* IPS Events
+* Faults
+* User Activity (audit logs)
+
+*config Syslog IPS Events:*
+1. on McAfee NSM go to Manager -> Setup -> Notification -> IPS Events -> syslog.
+2. click *YES* on "Enable Syslog Notification".
+3. bellow click on *+* and add Target server. 
+   * if you not configured target server alick add neer the "Target Server" and  fill the target server details.
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:15]:
```suggestion
   If you do not have a configured target server, click **Add** near Target Server and  fill in the target server details.
```

File: Packs/McAfeeNSM/README.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+   ![link](https://raw.githubusercontent.com/demisto/content/2063d324e6515a85b484705df5e4d153425e5110/Packs/McAfeeNSM/doc_imgs/nsm_add_target_server.png)
+4. on Facility select "Log Alert (note 1)".
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:17]:
```suggestion
4. In Facility, select **Log Alert (note 1)**.
```

File: Packs/McAfeeNSM/README.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+5. Severity mapping - not touch, need to be:
+ ![link](https://raw.githubusercontent.com/demisto/content/95eff3fe52a33695a10a76209cf8f2c4edbc185f/Packs/McAfeeNSM/doc_imgs/nsm_ips_severity_mapping.png)
+6. click on the Checkbox to enable "Notify for all Alerts" and click save.
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:20]:
```suggestion
6. Click the Notify for all Alerts checkbox.
7. Click **Save**.
```

File: Packs/McAfeeNSM/README.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+
+*config Syslog Faults Events:*
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:22]:
```suggestion
Configure Syslog Faults Events:
```

File: Packs/McAfeeNSM/README.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+1. on McAfee NSM go to Manager -> Setup -> Notification -> Faults -> syslog.
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:23]:
```suggestion
1. In McAfee NSM, go to **Manager** > **Setup**> **Notification** > **Faults** > **syslog**.
```

File: Packs/McAfeeNSM/README.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+2. click *YES* on "Enable Syslog Notification".
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:24]:
```suggestion
2. In Enable Syslog Notification, click **YES**.
```

File: Packs/McAfeeNSM/README.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+3. fill the fields "Server Name or IP Address" and "Port".
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:25]:
```suggestion
3. Fill in the "Server Name or IP Address" and "Port" fields .
```

File: Packs/McAfeeNSM/README.md [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+4. on *facilities* dropdown select "Security/authorization (code 4)".
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:26]:
```suggestion
4. In the Facilities dropdown, select **Security/authorization (code 4)**.
```

File: Packs/McAfeeNSM/README.md [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+* User Activity (audit logs)
+
+*config Syslog IPS Events:*
+1. on McAfee NSM go to Manager -> Setup -> Notification -> IPS Events -> syslog.
+2. click *YES* on "Enable Syslog Notification".
+3. bellow click on *+* and add Target server. 
+   * if you not configured target server alick add neer the "Target Server" and  fill the target server details.
+   ![link](https://raw.githubusercontent.com/demisto/content/2063d324e6515a85b484705df5e4d153425e5110/Packs/McAfeeNSM/doc_imgs/nsm_add_target_server.png)
+4. on Facility select "Log Alert (note 1)".
+5. Severity mapping - not touch, need to be:
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:18]:
```suggestion
5. In Severity mapping, leave the configuration as it.  It should be:
```

File: Packs/McAfeeNSM/README.md [hunk 14]
```diff
@@ -0,0 +1,71 @@
...
+ ![link](https://raw.githubusercontent.com/demisto/content/95eff3fe52a33695a10a76209cf8f2c4edbc185f/Packs/McAfeeNSM/doc_imgs/nsm_ips_severity_mapping.png)
+6. click on the Checkbox to enable "Notify for all Alerts" and click save.
+
+*config Syslog Faults Events:*
+1. on McAfee NSM go to Manager -> Setup -> Notification -> Faults -> syslog.
+2. click *YES* on "Enable Syslog Notification".
+3. fill the fields "Server Name or IP Address" and "Port".
+4. on *facilities* dropdown select "Security/authorization (code 4)".
+5. severity mapping not touch, will be like:
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:27]:
```suggestion
5. In Severity mapping, leave the configuration as it.  It should be:
```

File: Packs/McAfeeNSM/README.md [hunk 15]
```diff
@@ -0,0 +1,71 @@
...
+    ![link](https://raw.githubusercontent.com/demisto/content/53399299a79f6d8323502c6489c02b87a8720a7b/Packs/McAfeeNSM/doc_imgs/nsm_faults_severity_mapping.png)
+6. on *Forward Faults* dropdown select "informational and above".
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:29]:
```suggestion
6. In the Forward Faults dropdown, select **informational and above**.
```

File: Packs/McAfeeNSM/README.md [hunk 16]
```diff
@@ -0,0 +1,71 @@
...
+7. on *Message Preference* click on the checkbox "Syslog default" and click save.
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:30]:
```suggestion
7. In Message Preference, click the Syslog default checkbox.
8. Click **Save**.
```

File: Packs/McAfeeNSM/README.md [hunk 17]
```diff
@@ -0,0 +1,71 @@
...
+
+*config Syslog User Activity (audit logs) Events:*
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:32]:
```suggestion
Configure Syslog User Activity (audit logs) Events:
```

File: Packs/McAfeeNSM/README.md [hunk 18]
```diff
@@ -0,0 +1,71 @@
...
+1. on McAfee NSM go to Manager -> Setup -> Notification -> User Activity -> syslog.
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:33]:
```suggestion
1. In McAfee NSM, go to **Manager** > **Setup**> **Notification** > **User Activity** > **syslog**.
```

File: Packs/McAfeeNSM/README.md [hunk 19]
```diff
@@ -0,0 +1,71 @@
...
+2. click *YES* on "Enable Syslog Notification".
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:34]:
```suggestion
2. In Enable Syslog Notification, click **YES**.
```

File: Packs/McAfeeNSM/README.md [hunk 20]
```diff
@@ -0,0 +1,71 @@
...
+3. fill *Server Name or IP Address* and *port*.
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:35]:
```suggestion
3. Fill in the "Server Name or IP Address" and "Port" fields .
```

File: Packs/McAfeeNSM/README.md [hunk 21]
```diff
@@ -0,0 +1,71 @@
...
+5. severity mapping not touch, will be like:
+    ![link](https://raw.githubusercontent.com/demisto/content/53399299a79f6d8323502c6489c02b87a8720a7b/Packs/McAfeeNSM/doc_imgs/nsm_faults_severity_mapping.png)
+6. on *Forward Faults* dropdown select "informational and above".
+7. on *Message Preference* click on the checkbox "Syslog default" and click save.
+
+*config Syslog User Activity (audit logs) Events:*
+1. on McAfee NSM go to Manager -> Setup -> Notification -> User Activity -> syslog.
+2. click *YES* on "Enable Syslog Notification".
+3. fill *Server Name or IP Address* and *port*.  
+4. select Protocol on *Protocol* dropdown.
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:36]:
```suggestion
4. In the Protocol dropdown, select **Protocol**.
```

File: Packs/McAfeeNSM/README.md [hunk 22]
```diff
@@ -0,0 +1,71 @@
...
+5. on *Facilities* dropdown select "Log Alert (note 1)".
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:37]:
```suggestion
5. In the Facilities dropdown, select **Log Alert (note 1)**.
```

File: Packs/McAfeeNSM/README.md [hunk 23]
```diff
@@ -0,0 +1,71 @@
...
+6. on severity mapping not touch, will be like:
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:38]:
```suggestion
6. In Severity mapping, leave the configuration as it.  It should be:
```

File: Packs/McAfeeNSM/README.md [hunk 24]
```diff
@@ -0,0 +1,71 @@
...
+    ![link](https://raw.githubusercontent.com/demisto/content/53399299a79f6d8323502c6489c02b87a8720a7b/Packs/McAfeeNSM/doc_imgs/nsm_audit_severity_mapping.png)
+7. on *Forward audit* dropdown select "Allow All Auditlogs".
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:40]:
```suggestion
7. In the Forward audit dropdown, select **Allow All Auditlogs**.
```

File: Packs/McAfeeNSM/README.md [hunk 25]
```diff
@@ -0,0 +1,71 @@
...
+7. on *Message Preference* click on the checkbox "Syslog default" and click save.
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:41]:
```suggestion
7. In Message Preference, click the Syslog default checkbox.
8. Click **Save**.
```

File: Packs/McAfeeNSM/README.md [hunk 26]
```diff
@@ -0,0 +1,71 @@
...
+
+## Event Time configuration
+
+By Default on Fault and IPS events (syslog) not have event time, to add event time do the followin steps:
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:45]:
```suggestion
By default, on Fault and IPS events (syslog) do not have an event time. To add an event time, perform the following:
```

File: Packs/McAfeeNSM/README.md [hunk 27]
```diff
@@ -0,0 +1,71 @@
...
+*IPS Events*
+1. on McAfee NSM go to Manager -> Setup -> Notification -> IPS Events -> syslog.
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:47]:
```suggestion
1. In McAfee NSM, go to **Manager** > **Setup**> **Notification** > **IPS Events** > **syslog**.
```

File: Packs/McAfeeNSM/README.md [hunk 28]
```diff
@@ -0,0 +1,71 @@
...
+2. choose target server and click on the pencil (edit)
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:48]:
```suggestion
2. Choose the target server and click the pencil (edit).
```

File: Packs/McAfeeNSM/README.md [hunk 29]
```diff
@@ -0,0 +1,71 @@
...
+3. on message part add in the end of the string " at  $IV_ATTACK_TIME$"
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:49]:
```suggestion
3. In the message part, add in the end of the string " at  $IV_ATTACK_TIME$".
```

File: Packs/McAfeeNSM/README.md [hunk 30]
```diff
@@ -0,0 +1,71 @@
...
+4. click save
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:50]:
```suggestion
4. Click **Save**.
```

File: Packs/McAfeeNSM/README.md [hunk 31]
```diff
@@ -0,0 +1,71 @@
...
+
+*Syslog Faults*
+1. on McAfee NSM go to Manager -> Setup -> Notification -> Faults -> syslog.
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:53]:
```suggestion
1. In McAfee NSM, go to **Manager** > **Setup**> **Notification** > **Faults** > **syslog**.
```

File: Packs/McAfeeNSM/README.md [hunk 32]
```diff
@@ -0,0 +1,71 @@
...
+2. on message preference click edit and add to the end of the message " at  $IV_FAULT_TIME$"
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:54]:
```suggestion
2. In Message Preference, click edit and add to the end of the message " at  $IV_FAULT_TIME$".
```

File: Packs/McAfeeNSM/README.md [hunk 33]
```diff
@@ -0,0 +1,71 @@
...
+3. click save
```
Comment (ShirleyDenkberg) [inline Packs/McAfeeNSM/README.md:55]:
```suggestion
3. Click **Save**.
```

**Comment by ShirleyDenkberg** [comment]
@evisochek Doc review completed.

**Review by evisochek — APPROVED** [review]

**Pull request merged**